Core/Frameworks/BaikalAdmin/Controller/User/Calendars.php

```diff
@@ -269,4 +270,15 @@ function linkHome() {
             "user" => $this->currentUserId(),
         ]);
     }
+
+    /**
+     * Generate a link to the CalDAV/CardDAV URI of the calendar.
+     *
+     * @param \Baikal\Model\Calendar $calendar
+     *
+     * @return string Calender DAV URI
+     */
+    protected function getDavUri(\Baikal\Model\Calendar $calendar) {
+        return BAIKAL_DAV_BASEURI . 'calendars/' . $this->oUser->get('username') . '/' . $calendar->get('uri') . '/';
```

Comment:
`BAIKAL_DAV_BASEURI` is no longer used since we migrated to a new config format (see #899). This should use the value from the yml file instead. The PR as-is throws an exception (`Use of undefined constant BAIKAL_DAV_BASEURI`).

Reply:
Ah, snap. Hoped a rebase would be sufficient. Thanks for the check. According to the [linked PR](https://github.com/ByteHamster/Baikal/blob/97360514856f25e4d1ebbe226cdaa828cebe05ea/html/dav.php#L67) it seems sufficient to replace `BAIKAL_DAV_BASEURI` with `PROJECT_BASEURI . 'dav.php/'`?